src/utils/escapeRegex.js

```diff
@@ -0,0 +1,3 @@
+const escapeRegex = str => str.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&')
```

Comment:
Holy crap... Care to add a comment for people like me who don't just look at this and immediately know what is going on?